examples/lke_echo_server/providers.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    linode = {
+      source = "linode/linode"
+      version = "1.18.0"
```

Comment:
Remove version references for our provider. This means less work for us when we do releases